packages/manager/apps/procedures/src/pages/create/Create.spec.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { describe, expect, it, vi } from 'vitest';
+import { render } from '@testing-library/react';
+import Create from '@/pages/create/Create.page';
+import { LegalPolicyLinkByLanguage } from '@/constants';
+
+const user = {
+  legalForm: 'other',
+  subsidiary: 'FR',
+  language: 'en_CA',
+};
+
+vi.mock('react-i18next', () => ({
+  useTranslation: () => ({
+    t: (translationKey: string, args: any) =>
+      `${translationKey} ${JSON.stringify(args)}`,
+    i18n: {
+      language: user.language,
+    },
+  }),
+}));
+
+vi.mock('@/context/User/useUser', () => ({
+  default: () => ({
+    user,
+  }),
+}));
+
+describe('Create.page', () => {
+  it('should have the correct link for FR user', async () => {
```

Comment:
**niptick** : you can create only one test and set all your different cases with `it.each`